Paper.tex

```diff
@@ -42,7 +42,10 @@
 \usepackage[autostyle]{csquotes}
 \MakeOuterQuote{"}
 
-\input{Version.tex}
```

Comment:
Why is this removing `Version.tex`?

Reply:
Version tracking is moved to the more generic `Options.tex`.

New line: https://github.com/ethereum/yellowpaper/pull/582/files/7a9a385a4e89303a9351c27939cfa0c698628d0c#diff-9f702e1491c55da9d76a68d651278764R48

I found this as a best practice from Tex StackExchange.

Source:
https://tex.stackexchange.com/a/1502/140752

Reply:
I was looking at the build script but couldn't find the line putting the version into `Options.tex`. I do see it now.